lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/session-timeout.yaml

```diff
@@ -0,0 +1,21 @@
+# Entry points
+
+CORE_SESSION_TIMEOUT:
+  events:
+    next:
+      targetState: TIMEOUT_UNRECOVERABLE_PAGE
+
+# Journey states
+
+TIMEOUT_UNRECOVERABLE_PAGE:
+  response:
+    type: page
+    pageId: pyi-timeout-unrecoverable
```

Comment:
FYI @RogVanGDS is working on [PYIC-4432](https://govukverify.atlassian.net/browse/PYIC-4432) to make the pyi-timeout page dynamic so there'll probably be a clash here